code/00_setup-environment.R

```diff
@@ -74,9 +103,8 @@ external <-  c(paste0("V", 0, 1:9), paste0("V", 10:99),
 
 
 ### 5 - Read in lookup files ----
-## TO DO - do we need all three lookups?
 
-postcode <- 
+postcode <- function(){
   
   read_rds(glue("{filepath}lookups/Unicode/Geography/",
```

Comment:
I'm thinking can we automate reading in the .rds file so that don't have to manually change the file name when SPD name changes every 6 months? Can do something like this (you can change to use glue function)?
```
spd <- list.files("//Isdsf00d03/cl-out/lookups/Unicode/Geography/Scottish Postcode Directory", pattern = ".rds")
postcode <- read_rds(paste0("//Isdsf00d03/cl-out/lookups/Unicode/Geography/Scottish Postcode Directory/", spd))
```